packages/editor/src/client/atoms/emojiPicker.tsx

```diff
@@ -4,14 +4,14 @@ import './emojiPicker.less'
 import {Picker, BaseEmoji} from 'emoji-mart'
 
 interface EmojiPickerProps {
-  doWithEmoji: (emoji: string) => void
+  withEmoji: (emoji: string) => void
```

Comment:
for consistent naming with colorPicker component

Reply:
same argument here about `setEmoji`